plugins/woocommerce-admin/client/two-column-tasks/task-list.tsx

```diff
@@ -229,7 +237,14 @@ export const TaskList: React.FC< TaskListProps > = ( {
 		return <div className="woocommerce-task-dashboard__container"></div>;
 	}
 
+	const hasTaskListCompletionSlotFills = Boolean(
+		taskListCompletionSlot?.fills?.length
+	);
+
 	if ( isComplete && keepCompletedTaskList !== 'yes' ) {
+		if ( hasTaskListCompletionSlotFills ) {
+			return <TaskListCompletionSlot />;
```

Comment:
@rjchow Apologies for the late comment on this one, can we pass `hideTasks`, `keepTasks`, and `customerEffortScore` to the Fill via `fillSlot` parameter? This would ease our fill implementation later since those will definitely be used in the new UI

Reply:
done! thanks for catching that!

Reply:
![image](https://user-images.githubusercontent.com/27843274/213674557-ede56498-f9b9-40c9-a14c-78c063c1ffab.png)